content/en/security/cloud_security_management/iac_scanning.md

```diff
@@ -0,0 +1,39 @@
+---
+title: IaC Scanning
+private: true
+---
+
+<div class="alert alert-info">Static Infrastructure as Code (IaC) scanning is in private beta.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Static Infrastructure as Code (IaC) scanning is in Preview.</div>
```
Just noting we can say Preview as long as it doesn't say Private Beta in the UI.